arch/arm/core/aarch32/Kconfig

```diff
@@ -255,14 +255,16 @@ choice
 
```

Comment:
"Add the installed interface source files exported by the TF-M build system compile them in the zephyr build system."

Language.

Reply:
nvm. It's in a fromtree, so it's too late.